presto-main/src/test/java/com/facebook/presto/cost/ColumnStatisticsAssertion.java

```diff
@@ -91,4 +91,17 @@ public ColumnStatisticsAssertion distinctValuesCountUnknown()
                 "expected unknown distinctValuesCount but got " + statistics.getOnlyRangeColumnStatistics().getDistinctValuesCount().getValue());
         return this;
     }
+
+    public ColumnStatisticsAssertion dataSize(double expected)
+    {
+        assertEquals(statistics.getOnlyRangeColumnStatistics().getDataSize().getValue(), expected, "dataSize mismatch");
```

Comment:
This did not come as a problem yet.
If it appears we will add that.
Probably following http://floating-point-gui.de/errors/comparison